web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Actions/page/index.tsx

```diff
@@ -11,45 +12,54 @@ type ActionsPageProps = {
 // TODO: Ad TWK Lausanne font
 export const ActionsPage = ({ searchParams }: ActionsPageProps) => {
   const createAction = searchParams?.createAction;
+  const alpha = true;
   if (createAction) {
     return <CreateActionModal />;
```

Comment:
@andy-t-wang Maybe it's better to show/hide this modal with styles (opacity, visibility, pointer-events) instead of changing return of the whole component.

Reply:
should we do this with the create action modal as well?

Reply:
just did it